Documentation/FrameworkOverview.md

```diff
@@ -8,36 +8,33 @@ learning about new modules and finding more specific documentation.
 For examples and help understanding how to use RAC, see the [README][] or
 the [Design Guidelines][].
 
-## Streams
+## Signals
+
+A **signal**, represented by the [Signal][] class, is any series of objects.
 
-A **stream**, represented by the [RACStream][] abstract class, is any series of
-object values.
+<!-- TODO: This is something I don't know, is it still true for Signal?  -->
+<!-- Signals are [monads][]. Among other things, this allows complex operations to be
+built on a few basic primitives (`-bind:` in particular). [Signal][] also
+implements the equivalent of the [Monoid][] and [MonadZip][] typeclasses from
+[Haskell][]. -->
 
 Values may be available immediately or in the future, but must be retrieved
 sequentially. There is no way to retrieve the second value of a stream without
 evaluating or waiting for the first value.
```

Comment:
This paragraph doesn't apply anymore. Signals are push driven and consumers will miss events if not observing when the event is sent.